jatszohaz/jatszohaz/views.py

```diff
@@ -183,3 +187,26 @@ def get(self, request, token, *args, **kwargs):
         login(request, user)
         messages.info(request, _("Logged in as user %s.") % str(user))
         return redirect("/")
+
+
+class NewCommentView(LoginRequiredMixin, FormView):

```

Comment:
Itt a `LoginRequiredMixin` nem jó, mert ha jól gondolom csak a körtagoknak akarunk ehhez jogot adni. Így bármelyik bejelentkezett felhasználó tudna új kommentet írni (igaz, kicsit trükközve, mert maga a form nem jelenik meg, de tudna).

Helyette én így csinálnám, így csak azok férnek hozzá, akiknek van `jatszohaz.view_all` jogosultságuk, ami csak körtagoknak van.
```suggestion
class NewCommentView(PermissionRequiredMixin, FormView):
    permission_required = 'jatszohaz.view_all'
    raise_exception = True
```